WooCommerce/Classes/ViewRelated/Products/ProductsViewController.swift

```diff
@@ -439,21 +439,6 @@ private extension ProductsViewController {
             requestAndShowErrorTopBannerView()
             return
```

Comment:
Since this is now the only thing this method does, what do you think of removing the `guard` statement (and updating the method documentation to reflect what it does now)? This could be a regular check like:

```swift
    /// Displays an error banner if there is an error loading products data.
    ///
    func showTopBannerViewIfNeeded() {
        if hasErrorLoadingData {
            requestAndShowErrorTopBannerView()
        }
    }
```